src/Common/Factory/AbstractFactoryRule.php

```diff
@@ -8,7 +8,7 @@
 
 abstract class AbstractFactoryRule extends SprykerAbstractRule
 {
-    const PATTERN_ZED_FACTORY = '/\\\\*\\\\.+\\\\.+\\\\.+Factory$/';
+    const PATTERN_ZED_FACTORY = '/\\\\*\\\\.+\\\\.+\\\\Business\\\\[^\\\\]+Factory$/';
```

Comment:
[^\\\\] could also be [A-Za-z0-9] right?

Reply:
yes, might be. Needed?

Reply:
I think it will be fine as it is now.